.github/actions/minikube-setup/action.yml

```diff
@@ -5,18 +5,15 @@ runs:
   using: "composite"
   steps:
     - name: Setup Minikube
-      shell: bash
-      run: |
-        wget --no-verbose https://github.com/kubernetes/minikube/releases/download/v1.25.1/minikube-linux-amd64
-        sudo cp minikube-linux-amd64 /usr/local/bin/minikube
-        sudo chmod 755 /usr/local/bin/minikube
-        sudo apt-get install -y conntrack socat
```

Comment:
@tenzen-y looks like we still need to install socat, there is a testing depending on port forward.
```
E1211 06:49:51.448049   15133 portforward.go:400] an error occurred forwarding 9088 -> 9088: error forwarding port 9088 to pod f1e633307b22ea0960e456dd9ad12de6d74135bc9f4c2751f66fb402bba2f768, uid : unable to do port forwarding: socat not found
```

Reply:
Oops...
Thanks for letting me know :)

Reply:
Done.